tests/scikit_learn/etl/test_split.py

```diff
@@ -83,6 +83,70 @@ def test_split_randomness():
                                    'petallength', 'petalwidth'], size=slice_size))
 
 
+def test_split_fail_by_slice_size():
+    slice_size = 1  # No need to split in this case... is this value allowed in the GUI?
```

Comment:
Yes, this value is allowed in GUI because the backend considers 0.5 as default value. So, your test should pass if:

len(result['split_1_task_1']) == len(result['split_2_task_1']) == len(df[1])/2 == slice_size / 2 == 5. 
I don't know the behavior if there is an odd number of rows.